FutureNNAimbot/Program.cs

```diff
@@ -175,19 +188,18 @@ static void Main(string[] args)
                 }
                 if (yoloWrapper.DetectionSystem.ToString() != "GPU")
                 {
-                    MessageBox.Show("No GPU card detected. Exit...");
+                    MessageBox.Show("No GPU card detected. Exiting...");
                     Process.GetCurrentProcess().Kill();
                 }
                 objects = File.ReadAllLines($"trainfiles/{game}.names");
             }
             else
             {
                 trainingMode = true;
-                MessageBox.Show($"Looks like you haven't configured configs for game {game}... Let's go to train Neural Network! :)\n Prepairing files for training....");
+                MessageBox.Show($"Looks like you haven't configured settings for the game {game}... Let's train the Neural Network! :)\n Prepairing files for training....");
```

Comment:
It looks like you have not configured the settings for the game {game}... Let's now train the Neural Network! :)\n Repairing files for training mode....");